pages/ressources.js

```diff
@@ -0,0 +1,26 @@
+import marked from 'marked'
+import Page from '../components/Page'
+
+let text = `
+# Ressources for learning JavaScript
+
+We compiled a list of free ressources we think will help you learn JavaScript:
+
+<h3> 1. freeCodeCamp </h3>
+
+FreeCodeCamp is an interactive online learning platform designed to teach you everything web development related. 
+JavaScript is the most popular language to learn on the platform and as such they have multiple tutorials and assignments
+you can take for free.
+
+<a href="https://www.freecodecamp.org/" target="_blank">Check it out here</a>
```

Comment:
"Check it out here" is not a good anchor text either for visitors, seo or screenreaders 😄

Reply:
What do you think will be a better anchor text? :)

Reply:
"Check out freecodecamp"